ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitter.scala

```diff
@@ -151,11 +151,12 @@ private[kvutils] class TransactionCommitter(
 
   /** Validate ledger effective time and the command's time-to-live. */
   private[committer] def validateLedgerTime: Step =
-    (commitContext, transactionEntry) =>
-      commitContext.getRecordTime
-        .map { recordTime =>
-          val (_, config) = getCurrentConfiguration(defaultConfig, commitContext.inputs, logger)
-          val timeModel = config.timeModel
+    (commitContext, transactionEntry) => {
+      val (_, config) = getCurrentConfiguration(defaultConfig, commitContext.inputs, logger)
+      val timeModel = config.timeModel
+
+      commitContext.getRecordTime match {
```

Comment:
If you are pattern matching on an `Option` always have `case Some(_)` and `case None`. Otherwise, you could just keep the `.map` + `.getOrElse` that was used here.

Reply:
Yes, made the `None` branch explicit as also suggested by @andreaslochbihler-da.